scripts/byron-to-alonzo/simple-tx.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+# This script creates, signs, and submits a simple transaction in the Alonzo era.
+# We simply generate a new user and send him some money from a pre-build utxo account.
```

Comment:
We can leave out the "from a pre-build utxo account.".

Reply:
:+1: